src/ripple/ledger/View.h

```diff
@@ -341,6 +341,12 @@ bool amendmentRIPD1141 (NetClock::time_point const closeTime);
 NetClock::time_point const& amendmentRIPD1141SoTime ();
 bool amendmentRIPD1141 (NetClock::time_point const closeTime);
 
+NetClock::time_point const& amendmentRIPD1274SoTime ();
+bool amendmentRIPD1274 (NetClock::time_point const closeTime);
+
+NetClock::time_point const& amendmentRIPD1274SoTime ();
+bool amendmentRIPD1274 (NetClock::time_point const closeTime);
```

Comment:
I think the declarations on these two lines exactly duplicate the declarations on the previous two lines.  Or did I miss something?